File: 1-dic/1.dart
import 'dart:io';

main() {
  int res = 0;
  while(true) {
    String s = stdin.readLineSync();

    if(s == '')
      break;

    res += int.parse(s) ~/ 3 - 2;  
  }

  print('res --> $res');
}

File: 2018/01/1.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    freopen("input/in.txt","r",stdin);
    freopen("output/out1.txt", "w",stdout);

    int n, res = 0;
    while(cin >> n){ res += n; }

    cout << res;

    return 0;
}
